R/calcEDGETrData.R

```diff
@@ -15,7 +15,7 @@ calcEDGETrData <- function() {
   ## for all "default" SSP variants we ship the whole zoo of EDGE-T scenarios
   edgetScenarios <- strsplit(cartesian(
     c("SSP1", "SSP2", "SSP5", "SSP2EU", "SDP"),
-    c("ElecEra", "HydrHype", "Mix", "ConvCase")), split=".", fixed=TRUE)
```

Comment:
why mix1-4, are there any meaningful names possible? maybe for later...

Reply:
those are intended as a temporary solution, so I used these dummy names (I agree with you)